src/ParodusInternal.h

```diff
@@ -31,6 +31,9 @@
 
 #include "wss_mgr.h"
 #include "parodus_log.h"
+#ifdef PARODUS_LOGGER
+#include <rdk_debug.h>
```

Comment:
Hi,
rdk_debug.h was added here as we were getting below error while building in yocto:

RDKB2111/build-arrisxb3/tmp/work/arm1176jzstb-rdk-linux-uclibceabi/parodusarm/gitAUTOINC+3f52ba2888-r0/git/src/main.c:59:9: error: implicit declaration of function 'rdk_logger_init' [-Werror=implicit-function-declaration]
|          rdk_logger_init("/etc/debug.ini");
|          ^
| cc1: all warnings being treated as errors
| make[2]: *** [src/CMakeFiles/parodus.dir/main.c.o] Error 1
| make[1]: *** [src/CMakeFiles/parodus.dir/all] Error 2
| make: *** [all] Error 2

In parodus all the warnings are treated as errors and facing this issue even though rdk-logger is integrated using parodus recipe file. With this changes, dependency issues are solved and it is working fine. But we may have to think about a better solution to solve this "rdk_debug.h" dependency.